src/entities/log.ts

```diff
@@ -10,27 +10,39 @@ import {
 import { Application } from './application';
 import { AccessToken } from './accessToken';
 
+export const LogStatus = {
+  PENDING: 'pending',
+  DONE: 'done',
+  FAILED: 'failed',
+};
+
 @Entity()
 export class Log extends BaseEntity {
   @PrimaryGeneratedColumn()
   readonly id: number;
 
   @Column()
-  serviceName: string;
+  url: string;
+  @Column()
+  method: string;
+  @Column()
+  scope?: string;
   @Column()
   status: string;
   @Column()
-  error: string;
+  statusCode?: number;
+  @Column()
+  error?: string;
   @Column()
   trackId: string;
   @Column()
-  result: string;
+  result?: string;
 
   @Index()
   @ManyToOne(_type => Application)
-  application: Application;
+  application?: Application;
   @RelationId((log: Log) => log.application)
-  applicationId: number;
+  applicationId?: number;
```

Comment:
As a general rule for all entities we need to add createdAt and updatedAt columns.

I see the trackId has a date but let's avoid doing extra string splits and calculation if we don't need to, so lets save the createdAt on it's own separated column.